lix/packages/sdk/src/newLix.ts

```diff
@@ -34,15 +40,19 @@ export async function newLixFile(): Promise<Blob> {
         operation TEXT NOT NULL,
         value TEXT,
         meta TEXT,
-        commit_id TEXT
-        ) strict;
+        commit_id TEXT,
+        conflict TEXT
+      ) strict;
         
       CREATE TABLE 'commit' (
         id TEXT PRIMARY KEY,
         user_id TEXT NOT NULL,
+        parent_id TEXT NOT NULL,
         description TEXT NOT NULL,
-        zoned_date_time TEXT NOT NULL
+        created TEXT DEFAULT CURRENT_TIMESTAMP NOT NULL
```

Comment:
Why created as name instead of timestamp?

Reply:
@samuelstroschein  not sure what you mean, a name is suppossed to explain what the value is not its datatype. a timestamp could be created modified or any other event of a lifetime of a record.